devel/clavus/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo  1.0
+
+github.setup        peter-mbx clavus 0.0.1
+
+categories          devel
+revision            0
+
+maintainers         {gmail.com:pietro.mobrici} openmaintainer
+license             MIT
+
+description         Clavum Lateris
```

The `description` and `long_description` for this port aren't helpful to anyone trying to find out more about it.

It was only after I visited the Github page for this project that I got some clue into its intended purpose.

I'd recommend being more clear here.

```suggestion
description         \
    Manage the configurations on your machine, allowing you to configure your environment \
    and load the necessary resources, like files and/or commands.
    
long_description     {*}${description}
```